test/new-e2e/tests/process/windows_test.go

```diff
@@ -5,149 +5,149 @@
-		awshost.WithAgentOptions(agentparams.WithAgentConfig(processCheckConfigStr), agentparams.WithSystemProbeConfig(systemProbeConfigStr)),
-	))
-
-	// Flush fake intake to remove payloads that won't have IO stats
-	s.Env().FakeIntake.Client().FlushServerAndResetAggregators()
-
-	assert.EventuallyWithT(t, func(collect *assert.CollectT) {
-		command := "& \"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent.exe\" status --json"
-		assertRunningChecks(collect, s.Env().RemoteHost, []string{"process", "rtprocess"}, true, command)
-	}, 1*time.Minute, 5*time.Second)
-
-	// s.Env().VM.Execute("Start-MpScan -ScanType FullScan")
-
-	var payloads []*aggregator.ProcessPayload
-	assert.EventuallyWithT(t, func(c *assert.CollectT) {
-		var err error
-		payloads, err = s.Env().FakeIntake.Client().GetProcesses()
-		assert.NoError(c, err, "failed to get process payloads from fakeintake")
-
-		// Wait for two payloads, as processes must be detected in two check runs to be returned
-		assert.GreaterOrEqual(c, len(payloads), 2, "fewer than 2 payloads returned")
-	}, 2*time.Minute, 10*time.Second)
-
-	assertProcessCollected(t, payloads, true, "MsMpEng.exe")
-}
-
-func (s *windowsTestSuite) TestManualProcessCheck() {
-	check := s.Env().RemoteHost.
-		MustExecute("& \"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent\\process-agent.exe\" check process --json")
-
-	assertManualProcessCheck(s.T(), check, false, "MsMpEng.exe")
-}
-
-func (s *windowsTestSuite) TestManualProcessDiscoveryCheck() {
-	// Skipping due to flakiness
-	// Responses with more than 100 processes end up being chunked, which fails JSON unmarshalling
-
-	check := s.Env().RemoteHost.
-		MustExecute("& \"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent\\process-agent.exe\" check process_discovery --json")
-	assertManualProcessDiscoveryCheck(s.T(), check, "MsMpEng.exe")
-}
-
-func (s *windowsTestSuite) TestManualProcessCheckWithIO() {
-	s.T().Skip("skipping due to flakiness")
-	// MsMpEng.exe process missing IO stats, agent process does not always have CPU stats populated as it is restarted multiple times during the test suite run
-	// Investigation & fix tracked in https://datadoghq.atlassian.net/browse/PROCS-3757
-
-	s.UpdateEnv(awshost.Provisioner(
-		awshost.WithEC2InstanceOptions(ec2.WithOS(os.WindowsDefault)),
-		awshost.WithAgentOptions(agentparams.WithAgentConfig(processCheckConfigStr), agentparams.WithSystemProbeConfig(systemProbeConfigStr)),
-	))
-
-	// Flush fake intake to remove payloads that won't have IO stats
-	s.Env().FakeIntake.Client().FlushServerAndResetAggregators()
-
-	check := s.Env().RemoteHost.
-		MustExecute("& \"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent\\process-agent.exe\" check process --json")
-
-	// Check stats for Datadog agent process as it has IO stats more reliably populated than MsMpEng.exe
-	agentExe := "\"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent.exe\""
-	assertManualProcessCheck(s.T(), check, true, agentExe)
-}
+//import (
```

Comment:
We need to bring the 🪟 tests back

For future reference you can just run the Linux tests, see the command I posted in the testing instructions https://github.com/DataDog/datadog-agent/pull/22799 where I only ran the Windows test suite

Reply:
Sorry, that wasn't meant to be checked in. 🤦